pkgs/development/python-modules/gerbonara/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, pytest
+, click
+, numpy
+, scipy
+, rtree
+}:
+
+buildPythonPackage rec {
+  pname = "gerbonara";
+  version = "v1.2.0";
+  src = fetchFromGitHub {
+    owner = "jaseg";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-VU4Of90YUPoLuiUpIDwSUfxQOoKChNbZE0klHkHEmaY=";
```

Comment:
```suggestion
  version = "1.2.0";

  src = fetchFromGitHub {
    owner = "jaseg";
    repo = "gerbonara";
    rev = "v${version}";
    hash = "sha256-VU4Of90YUPoLuiUpIDwSUfxQOoKChNbZE0klHkHEmaY=";
```

Reply:
Again, I see no reason to remove `pname`.

Reply:
#277994 has a few arguments for this

Reply:
Done.